js/sponsors/sponsorEffect.js

```diff
@@ -54,3 +54,8 @@ $(document).ready(function() {
 		}
 	});
 });
+
+$(window).ready(function(){
+	// Open 
+	$('.heading').first().click();
```

Comment:
Hey @DonaGraciete is this a hack? Why do we need this?

Reply:
Yes, we need because it was a requirement on the new changes, to have the first table open, and I don't want to wast time adapting the JS and adding Css for now, so I just make the container being open on page ready

Reply:
Ok, seems cool for now! ;)